CppImport/src/visitors/ClangAstVisitor.h

```diff
@@ -28,9 +28,14 @@
 
 #include "../cppimport_api.h"
 #include "../manager/TranslateManager.h"
+#include "../macro/MacroImportHelper.h"
 #include "../CppImportLogger.h"
 #include "../comments/CommentParser.h"
 
+namespace clang {
+	class ParentMap;
```

Comment:
You do not use this anywhere in the .h file. Either put it in the cpp file or remove it if it is not necessary.